### PR TITLE
MAT-8037: Add lastModifiedAt

### PR DESCRIPTION
**Pull request by ethankaplan**
## MADiE PR

Jira Ticket: [MAT-8037](https://jira.cms.gov/browse/MAT-8037)
(Optional) Related Tickets:

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included and Code coverage is verified.
* [ ] No extemporaneous files are included (i.e Complied files or testing results).
* [ ] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/main/java/gov/cms/madie/cqllibraryservice/services/AppConfigService.java [hunk 1]
```diff
@@ -40,6 +40,10 @@ public void refreshAppConfig() {
   }
 
   public boolean isFlagEnabled(MadieFeatureFlag flag) {
+    if (featureFlags == null) {
+      log.warn("Feature flags are not initialized. Returning false for flag: {}", flag);
+      return false;
+    }
```
Comment (adongare) [inline src/main/java/gov/cms/madie/cqllibraryservice/services/AppConfigService.java:46]: is this related or from some other ticket?

**Review by adongare — APPROVED** [review]

File: src/main/java/gov/cms/madie/cqllibraryservice/services/AppConfigService.java (hunk 1 above)
Comment (ethankaplan) [reply src/main/java/gov/cms/madie/cqllibraryservice/services/AppConfigService.java:46]: This was causing errors during my setup, if featureFlags failed to load it would just explode the whole of library service. Figured it'd be good to fix the bug
